### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Shewolf227**
In this PR, the following were implemented:

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:

Screenshot 2022-05-23 at 8 06 39 PM

- Use class-based components (All components should be class-based)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (Kingstalux) [inline src/App.js]: - [x] Great job so far 👍 As per the requirements, you are to use class-based components. Kindly convert your app from a function to a class 💯 
-
Annotation 2022-05-23 181959

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @SoniaKarungi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
149637855-7722d04c-b45d-4185-925f-7de047106461


## Highlights 👏 
✅ App works as expected 👍 
✅ Linters are passing 👌 
✅ Great pull request description 🔥 
✅ Professional README file 💯 

 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Shewolf227) [reply src/App.js]: OOps 😬 .. thanks for the catch @Kingstalux. I must have just concentrated on the Calculator component. On it!

**Review by gemmen29 — APPROVED** [review]
Hi @SoniaKarungi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**